Tests/Algorithm/Framework/PythonHelper.cs

```diff
@@ -0,0 +1,31 @@
+﻿/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+*/
+
+using System;
+
+namespace QuantConnect.Tests.Algorithm.Framework
+{
+    public static class PythonHelper
+    {
+        public static void SetDefaultPythonPath()
```

Comment:
Maybe this should be implemented as a namespace-less `[SetUpFixture]` so we don't have to invoke once per test class.

http://nunit.org/docs/2.5.5/setupFixture.html